crates/cli-support/src/descriptor.rs

```diff
@@ -149,18 +149,21 @@ impl Descriptor {
         }
     }
 
-    pub fn is_number(&self) -> bool {
+    /// Returns `Some` if this type is a number, and the boolean returned
```

Comment:
I think it'd be nice if it returned a custom `enum` rather than `Option<bool>`. I don't feel too strongly about this though.

Reply:
👍